purldb-toolkit/src/purldb_toolkit/purlcli.py

```diff
@@ -185,7 +219,7 @@ def construct_headers(
     headers_content["options"] = options
     headers_content["purls"] = purls
 
-    if command_name == "metadata" and unique:
+    if (command_name in ["metadata", "urls", "validate", "versions"]) and unique:
         for purl in normalized_purls:
```

Comment:
```suggestion
        for input_purl, normalized_purl in normalized_purls:
            if input_purl != normalized_purl:
                warnings.append(f"input PURL: '{input_purl}' normalized to '{normalized_purl}'")
```

Reply:
Done.  Thanks @JonoYang -- naming the pair of PURLs is a nice clarification and I'll remember this going forward.